Sources/Bindings/SFTP.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SFTP.swift
+//  Bindings
+//
+//  Created by Vladislav Alexeev on 6/20/18.
+//
+
+import Foundation
+import CSSH
+
+public class SFTP {
+    private let sftpSession: OpaquePointer
+    
+    public struct Permissions : OptionSet {
+        public let rawValue: UInt
+        public init(rawValue: UInt) {
+            self.rawValue = rawValue
+        }
+        
+        public static let read = Permissions(rawValue: 1)
+        public static let write = Permissions(rawValue: 2)
```

Comment:
Should these rawValues be bit shifted? `read = 1 << 0`, `write = 1 << 1`, and `execute = 1 << 2`?

Reply:
Good catch.